Formula/faunus.rb

```diff
@@ -56,9 +56,13 @@ def install
       "-S", ".", *std_cmake_args,
       "-DPYTHON_EXECUTABLE=#{venv_root}/bin/python"
     system "make", "install"
+    mv prefix/"bin/yason.py", prefix/"bin/yason.py.unwrapped"
+    (bin/"yason.py").write_env_script prefix/"bin/yason.py.unwrapped",
+      PYTHONPATH: "#{venv_root}/lib/python#{xy}/site-packages"
   end
 
   test do
+    system "#{bin}/yason.py #{share}/faunus/examples/minimal.yml | #{bin}/faunus --quiet"
```

Comment:
```suggestion
    pipe_output( "#{bin}/faunus --quiet", "#{bin}/yason.py #{share}/faunus/examples/minimal.yml")
```
Is there output here we can check?